examples/jsm/controls/FirstPersonControls.js

```diff
@@ -155,45 +155,45 @@ var FirstPersonControls = function ( object, domElement ) {
 
 		//event.preventDefault();
 
-		switch ( event.keyCode ) {
+		switch ( event.code ) {
```

Comment:
You have to update the `examples/js` versions of the controls, too. Remember that modules are generated from the `examples/js` files via `node utils/modularize.js`.